packages/oas/src/utils.ts

```diff
@@ -2,6 +2,8 @@ import findSchemaDefinition from './lib/find-schema-definition.js';
 import matchesMimeType from './lib/matches-mimetype.js';
 import { types as jsonSchemaTypes } from './operation/lib/get-parameters-as-json-schema.js';
 
-const supportedMethods = new Set(['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace']);
+const methods = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace'] as const;
+
+const supportedMethods: Set<(typeof methods)[number]> = new Set(methods);
```

Comment:
could honestly probably do away with this `Set`. the only place where we're using this is in `oas/src/index.ts` where we're doing a `.has()` call on it. could be refactored to an `.includes()` check.

Reply:
cool, will fix

Reply:
https://github.com/readmeio/oas/pull/933/commits/e1843963b5529f2e1c328439bdc19cd75ce9a73d